pynicotine/gtkgui/wishlist.py

```diff
@@ -86,14 +89,20 @@ def cell_edited_callback(self, widget, index, value, treeview, pos):
         if value and not value.isspace():
             self.remove_wish(old_value)
             self.add_wish(value)
+            self.list_view.set_cursor(self.store.get_path(self.wishes[value]))
```

Comment:
- Incorrect listitem selected after renaming an item.

Reply:
Perhaps this could be moved to `add_wish` instead?

```
        if wish not in self.wishes:
            self.wishes[wish] = self.store.insert_with_valuesv(-1, self.column_numbers, [wish])
            self.list_view.set_cursor(self.store.get_path(self.wishes[wish]))
```